src/app/seller-post-ad-form/imageProperty.ts

```diff
@@ -0,0 +1,16 @@
+export class ImageProperty
+{
+  //properties of html element 
+  addEditProperty = "";
+  imageDisplayValue =  "none";
+  CrossBtnValue = "none";
+  buttonName = "Add";
+  iconOfButton = "plus"; 
+  pictureContainerStyle = "1px solid lightgrey";
+  heroImage="none";
+  imageLoaderProperty = "none";
+
+  //store url of image return by upload image api
+  imageURL="";
```

Comment:
Please revisit all variable in this class.
Remove if not required.

Reply:
Its required